bin/build-package-containing-file.js

```diff
@@ -0,0 +1,39 @@
+const fs = require( 'fs' );
+const { spawnSync } = require( 'child_process' );
+
+const args = process.argv.slice( 2 );
+if ( args.length < 1 ) {
```

Comment:
nit: I'd recommend to use `yargs` or `commander` (already present as dependencies) to do the arguments handling.